### DISCUSSION TITLE
Configurator errors on Linux

### DISCUSSION THREAD

**Discussion by urrfaust** [shanteacontrols/OpenDeck#251] [General]
Hi,

I'm having trouble running the configurator on my browser (firefox version 120) and stand-alone as well on Linux. I'm using Ubuntu 23.10.
My midi device is correctly recognised.
lsusb output:

`Bus 002 Device 004: ID 1209:8472 Generic Shantea Controls OpenDeck`

dmesg:

`usb 2-1: new full-speed USB device number 4 using xhci_hcd
usb 2-1: New USB device found, idVendor=1209, idProduct=8472, bcdDevice= 0.01
usb 2-1: New USB device strings: Mfr=1, Product=2, SerialNumber=3
usb 2-1: Product: OpenDeck | BB-B1
usb 2-1: Manufacturer: Shantea Controls
usb 2-1: SerialNumber: 4150324238323516
cdc_acm 2-1:1.0: ttyACM0: USB ACM device
usb 2-1: Quirk or no altest; falling back to MIDI 1.0`

On firefox I get this:
 
![image](https://github.com/shanteacontrols/OpenDeck/assets/6085127/c8041fd2-8c46-4ad0-a7bc-0f81aaf03d9e)

I have included the console output too at the bottom. Somehow the browser never asked me any permission to run WebMidi, like it  did on Windows 10.
When I run ./OpenDeckConfigurator from the terminal I basically get the same screen

![image](https://github.com/shanteacontrols/OpenDeck/assets/6085127/37b218cd-51d2-455e-8d73-e8eca96c8543)

and I get the following messages:

`libva error: /usr/lib/x86_64-linux-gnu/dri/iHD_drv_video.so init failed
[4922:1128/191634.260087:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.`

I have read somewhere that I need to disbale GPU acceleration but I don't really know how.

Thanks for your input

**Comment by paradajz** [comment]
What about Chome?

**Reply by urrfaust** [reply]
Same result. I get a different error in the console:

`INITIAL HANDSHAKE TIMED OUT, RETRYING`

I also ran it from the terminal with:

`google-chrome --disable-gpu --disable-software-rasterizer`

as I found on the web. I got rid of the errors that were shown on firefox but I still get Webmidi connecting...

Must be something about Webmidi running on Linux.

**Comment by paradajz** [comment]
Ubuntu 23.10 ships with kernel 6.5 which broke WebMIDI output in Chromium browsers:
https://bugs.chromium.org/p/chromium/issues/detail?id=1499631&q=component%3ABlink%3EWebMIDI&can=2

It doesn't work for me as well, but it works fine in Firefox, though (also on Windows).

**Reply by paradajz** [reply]
I believe the issue is now fixed.
